### ISSUE TITLE
A participant can be an organizer in multiple challenges

### ISSUE THREAD

**Issue by spMohanty** [AICrowd/AIcrowd#886]
The participant model seems to simply store a single organizer id. I am assuming that means we cannot add a participant to multiple organizations.

**Comment by mridulnagpal** [comment]
@spMohanty We can create a model to link between organizations and participants to deal with this situation. But we should confirm on what to do before making changes to the model. 
What do you recommend?

**Comment by spMohanty** [comment]
@mridulnagpal  @nemaniarjun : Yes indeed. We cant have a separate field in the participant model link to a challenge, because a user can be an organizer in multiple organizations. Please go ahead with creating a model linking organizations and participants.

**Comment by nemaniarjun** [comment]
Can we assign this to @bkDJ, I am not good with rails db stuff :sweat_smile:
